src/tribler-gui/tribler_gui/event_request_manager.py

```diff
@@ -59,12 +59,22 @@ def __init__(self, api_port, api_key):
             NTFY.LOW_SPACE.value: self.low_storage_signal.emit,
             NTFY.REMOTE_QUERY_RESULTS.value: self.received_remote_query_results.emit,
             NTFY.TRIBLER_SHUTDOWN_STATE.value: self.tribler_shutdown_signal.emit,
-            NTFY.EVENTS_START.value: lambda _: None,
```

Comment:
Well, you don't have to rollback your key-related stuff. You only have to call `events_start_received` on `EVENTS_START`. I checked that, that was enough to solve the problem. So, it is better to keep the key thing in - we really want it.